.github/workflows/sim.yaml

```diff
@@ -43,6 +43,10 @@ jobs:
         - "sig-rsa validate-primary-slot direct-xip"
         - "sig-rsa validate-primary-slot ram-load multiimage"
         - "sig-rsa validate-primary-slot direct-xip multiimage"
+        - "ram-load enc-aes256-kw multiimage"
+        - "ram-load enc-aes256-kw sig-ecdsa-mbedtls multiimage"
+        - "ram-load enc-aes256-ec256 multiimage"
+        - "ram-load enc-aes256-x25519 multiimage"
```

Comment:
I think these 2 can be removed. These image encryption algs are already tested separately and we have
other feature sets to test ram-load with multiple encrypted images. I believe these don't have too much added value.
What do you think?

Reply:
I have no objections.